Shoko.Server/Providers/AniDB/UDP/AniDBUDPConnectionHandler.cs

```diff
@@ -114,10 +114,10 @@ private async Task InitInternal()
         }
 
         var settings = SettingsProvider.GetSettings();
-        ArgumentNullException.ThrowIfNull(settings.AniDb?.ServerAddress);
-        if (settings.AniDb.ServerPort <= 0) throw new ArgumentException("AniDB Server Port is invalid");
+        ArgumentNullException.ThrowIfNull(settings.AniDb?.UDPServerAddress);
+        if (settings.AniDb.UDPServerPort <= 0) throw new ArgumentException("AniDB Server Port is invalid");
```

Comment:
Would it be better to tweak the exception to specifically state that it's the `UDP` Server Port that's invalid given that ServerPort would be a different setting?

Reply:
This case will never hit.

https://dotnetfiddle.net/NnHiBB

```csharp
ushort a = -1;
```
is a compiler error.

Reply:
So the line can just be removed then?